srcpkgs/vlna/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'vlna'
+pkgname=vlna
+version=1.5
+revision=1
+build_style=gnu-configure
+short_desc="Program for adding non-breaking spaces to TeX and other files"
```

Comment:
Could we mention here that this follows czech typesetting conventions?
```suggestion
short_desc="Program for adding non-breaking spaces to TeX and other files"
```

Reply:
I don't think it can be done in an elegant way and also follow the rule of maximum 72 characters. It is used for Slovak (and perhaps other) typography as well, so it would make sense to keep the description ambiguous.

Here are few suggestions, though:
- "Program for adding Czech & Slovak non-breaking spaces to (TeX) files"
- "Adding Czech & Slovak non-breaking spaces to TeX and other files"
- "Adding Czech & Slovak typography non-breaking spaces to (TeX) files"

Reply:
I like the second one